bin/generate_core_lockfile.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
```

Comment:
Hard to tell on github, but does this have exec bit set?

Reply:
Yes, it's 755